.github/workflows/extensions.yml

```diff
@@ -297,13 +297,23 @@ jobs:
       RELEASES: ${{ needs.fetch-releases.outputs.releases }}
 
     steps:
+      - name: Generate token for GitHub App
+        id: generate-token
+        uses: actions/create-github-app-token@v2
+        with:
+          app-id: 1129585
+          private-key: ${{ secrets.POSIT_CONNECT_PROJECTS_PEM }}
+          permissions: |
+            contents: write
+            pull-requests: write
```

Comment:
I don't think you do anything with PRs with this workflow

```suggestion
```

Also, IIRC "with" arguments can't be objects like this. Can they?

Elsewhere we do

```
        with:
          app-id: 1129585
          private-key: ${{ secrets.POSIT_CONNECT_PROJECTS_PEM }}
          permission-contents: write
```